Makefile

```diff
@@ -15,7 +15,7 @@ ifeq ($(shell echo $(PARSER_URL) | grep '^[a-z][-+.0-9a-z]*://'),)
 endif
 endif
 
-TS ?= tree-sitter
+TS ?= npx tree-sitter-cli
```

Comment:
**Question:** Is there a requirement for using the `tree-sitter-cli` JS package over using the [Rust crate](https://crates.io/crates/tree-sitter-cli)?

Reply:
This change doesn't seem to be necessary. I've reverted it.